arches_lingo/src/arches_lingo/components/basic-search/SortAndFilterControls.vue

```diff
@@ -1,27 +1,30 @@
 <script setup lang="ts">
-import { ref } from "vue";
 import { useGettext } from "vue3-gettext";
-
-import Checkbox from "primevue/checkbox";
 import RadioButton from "primevue/radiobutton";
 
 const { $gettext } = useGettext();
 
-const filters = [
-    { value: "Concepts", label: $gettext("Concepts") },
-    { value: "Places", label: $gettext("Places") },
-    { value: "People", label: $gettext("People") },
-    { value: "Groups", label: $gettext("Groups") },
-    { value: "Periods", label: $gettext("Periods") },
-];
-const queryFilters = ref([]);
+const ORDER_MODE_UNSORTED = "unsorted";
+const ORDER_MODE_ALPHABETICAL = "alphabetical";
+const ORDER_MODE_REVERSE_ALPHABETICAL = "reverse-alphabetical";
 
 const sortOptions = [
-    { value: "Unsorted", label: $gettext("Unsorted") },
-    { value: "A to Z", label: $gettext("A to Z") },
-    { value: "Z to A", label: $gettext("Z to A") },
+    { value: ORDER_MODE_UNSORTED, label: $gettext("Unsorted") },
```

Comment:
I think the label "Unsorted" here may be confusing, particularly given the improvements here, maybe we should use something like this?:
```suggestion
    { value: ORDER_MODE_UNSORTED, label: $gettext("Best match") },
```